logger/src/lib.rs

```diff
@@ -502,6 +524,66 @@ impl Logger {
         Ok(())
     }
 
+    /// Performs the most basic steps in order to enable the logger to write to StdOut or StdErr
```

Comment:
Please keep the first line of the comment as a short summary, as for `init`. For instance, `Preconfigure the logger prior to initialization.`